xbee/tornado/tests/test_ieee.py

```diff
@@ -0,0 +1,817 @@
+#! /usr/bin/python
+"""
+test_ieee.py
+
+By Paul Malmsten, 2010
+pmalmsten@gmail.com
+
+Tests the XBee (IEEE 802.15.4) implementation class for XBee API compliance
+"""
+from xbee.tests.Fake import Serial
+from xbee.tornado.ieee import XBee
+from xbee.frame import APIFrame
+from xbee.python2to3 import intToByte, stringToBytes
+from tornado.testing import AsyncTestCase, gen_test
+import sys
+import traceback
+
+
+class InitXBee(AsyncTestCase):
+    """
+    Base initalization class
+    """
+
+    def setUp(self, *args, **kwargs):
+        """
+        Initialize XBee object
+        """
+        super().setUp(*args, **kwargs)
```

Comment:
PyCharm does not like this. It complains that super() is not supported by Python 2.7 ?